src/bindings/html/dom.js

```diff
@@ -77,19 +76,31 @@ export function translateMutations(view, langs, mutations) {
 
   const elements = [];
 
-  targets.forEach(target => target.childElementCount ?
-      elements.push(...getTranslatables(target)) : elements.push(target));
+  targets.forEach(target => {
+    if (target.childElementCount) {
+      elements.push(...getTranslatables(target));
+    } else {
+      if (target.hasAttribute('data-l10n-id')) {
```

Comment:
It all boils down to personal preference, and I'm okay with your approach.

I think the following is also quite readable and I like how it's a single expression.

```javascript
targets.forEach(target => target.childElementCount ?
  elements.push(...getTranslatables(target)) :
  target.hasAttribute('data-l10n-id') ?
    elements.push(target)) : undefined
```

Again, this is just personal preference.  r+ on your version! :)